README.md

```diff
@@ -17,3 +17,11 @@ http://localhost:4000/home
 http://localhost:4000/applications
 
 http://localhost:4000/userprofile
+
+
+# General feedback 
+- Setup linting with Prettier and Eslint. It makes the codebase so much cleaner :) 
+- Maybe include firebase-emulator setup in a npm-command?
+- Consider using store instead of recoil? 
```

Comment:
See this article: https://blog.logrocket.com/use-hooks-and-context-not-react-and-redux/